api

```diff
@@ -675,6 +675,73 @@ debian_ppa_installer() { #setup a PPA on a Debian distro. Arguments: ppa_name di
     apt_update || exit 1
   fi
 }
+
+terminal_manage() { # wrapper for the original terminal_manage function to terminal_mange_multi
+  #function to install/uninstall/update/refresh one app (or filelist) - uses a terminal and refreshes the app list
```

Comment:
Is there a backwards-compatibility reason you chose to keep terminal_manage and terminal_manage_multi? My first inclination would be to either add multiline functionality to terminal_manage or to rename the function, updating all scripts which call the previous name.

Reply:
Yes, read the commit description